lib/backends/sdc/containers.js

```diff
@@ -2309,14 +2309,26 @@ function getContainers(opts, callback) {
     assert.object(opts.app.vmapi, 'opts.app.vmapi');
     assert.number(opts.clientApiVersion, 'opts.clientApiVersion');
     assert.arrayOfObject(opts.images, 'opts.images');
+    assert.optionalString(opts.filters, 'opts.filters');
     assert.optionalObject(opts.log, 'opts.log');
     assert.string(opts.req_id, 'opts.req_id');
 
+    var filters;
     var log = opts.log || this.log;
     var pkgmapNtoU = {};
     var pkgmapUtoN = {};
     var vmapi = opts.app.vmapi;
 
+    if (opts.filters) {
+        filters = JSON.parse(opts.filters);
+        filters = utils.getNormalizedFilters({ log: log }, filters);
+        if (filters instanceof Error) {
+            callback(new errors.DockerError('invalid filters: ' + filters));
+            return;
+        }
+        log.debug('filters: ', filters);
```

Comment:
Still want this? If so, please this:

`log.debug({filters: filters}, 'getContainers: filters');`